### PR TITLE
Changes to iOS app

### PR DESCRIPTION
**Pull request by martijnjanssen**
App icons and favicon are still needed to fix the app.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <base href="2017">
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=no">
```
Comment (TimvdLippe) [inline index.html:6]: For accessibility this should not be changed

Comment (martijnjanssen) [reply index.html:6]: Hmm okay, makes sense. But there has to be another way to prevent zooming when the application is in a web-wrapper.

Comment (TimvdLippe) [reply index.html:6]: Normall you don't want to disallow. Your choice though :smile:

**Review by TimvdLippe — APPROVED** [review]

**Comment by TimvdLippe** [comment]
You can merge this one if all was fixed :smile:

**Pull request merged**